### PR TITLE
Add new messages for mobile rewards

### PR DESCRIPTION
**Pull request by maplant**

### CODE REVIEW

File: src/service/poc_mobile.proto [hunk 1]
```diff
@@ -115,6 +115,8 @@ message speedtest {
   uint64 timestamp = 4;
 }
 
+/// Deprecated reward message for mobile. See mobile_reward_share and
```
Comment (madninja) [inline src/service/poc_mobile.proto:118]: Actually declare as deprecated.. see https://stackoverflow.com/questions/52781727/how-can-i-deprecate-whole-message-in-protocol-buffers

Comment (maplant) [reply src/service/poc_mobile.proto:118]: good catch! Thanks

**Comment by madninja** [comment]
You'll also want to remove the unused ones.. follow @bbalser lead here

File: src/service/poc_mobile.proto [hunk 2]
```diff
@@ -130,6 +131,30 @@ message radio_reward_share {
   uint64 end_epoch = 6;
 }
 
+message radio_reward {
+  /// Public key of the owner of the radio
+  bytes owner_key = 1;
```
Comment (bbalser) [inline src/service/poc_mobile.proto:136]: Do we need `owner_key` for the new message?  Once we hit solana, the owner_key is no longer used.

Comment (maplant) [reply src/service/poc_mobile.proto]: removed

**Review by bbalser — APPROVED** [review]

**Pull request merged**